### ISSUE TITLE
created_on and modified_on are always empty in table when Hidden on Browse

### ISSUE THREAD

**Issue by darioseidl** [directus/app#2863]
Using v8.7.1. We have a collection, where we need to see the owner and created_on. Unchecking "Hidden on Browse" and "Hidden on Detail" for the created_on field, makes the column show up in the collection table, but all cells are empty.

```
MySQL [vrb]> select * from checklist_checked;
+----+-------+---------------------+-------------+---------------------+-----------+
| id | owner | created_on          | modified_by | modified_on         | checklist |
+----+-------+---------------------+-------------+---------------------+-----------+
| 14 |     4 | 2020-06-02 14:07:12 |           4 | 2020-06-02 14:07:12 |         1 |
| 15 |     4 | 2020-06-03 15:20:24 |           4 | 2020-06-03 15:20:24 |         1 |
| 16 |     4 | 2020-06-03 15:20:47 |           4 | 2020-06-03 15:20:47 |         1 |
+----+-------+---------------------+-------------+---------------------+-----------+
```
![Screenshot_2020-06-09_16-27-55](https://user-images.githubusercontent.com/4629505/84161288-68872300-aa6f-11ea-8803-931f7355b5c8.png)

It's the same for modified_on.

**Comment by rijkvanzanten** [comment]
@darioseidl I believe this was fixed in v8.7.2 🙂

**Comment by darioseidl** [comment]
Great, thanks!
